sonic-xcvrd/tests/test_xcvrd.py

```diff
@@ -443,6 +443,27 @@ def test_CmisManagerTask_handle_port_change_event(self):
         task.on_port_update_event(port_change_event)
         assert len(task.port_dict) == 1
 
+    
+    @patch('xcvrd.xcvrd.XcvrTableHelper')
+    def test_CmisManagerTask_get_configured_freq(self, mock_table_helper):
+        port_mapping = PortMapping()
+        task = CmisManagerTask(DEFAULT_NAMESPACE, port_mapping)
+        cfg_port_tbl = MagicMock()
+        cfg_port_tbl.get = MagicMock(return_value=(True, (('laser_freq', 193100),)))
+        mock_table_helper.get_cfg_port_tbl = MagicMock(return_value=cfg_port_tbl)
+        task.xcvr_table_helper.get_cfg_port_tbl = mock_table_helper.get_cfg_port_tbl
+        assert task.get_configured_laser_freq('Ethernet0') == 193100
+
+    @patch('xcvrd.xcvrd.XcvrTableHelper')
+    def test_CmisManagerTask_get_configured_tx_power(self, mock_table_helper):
```

Comment:
Similarly can we add failure case? 1. when configured tx power is out of range.

Reply:
This API simply get the configured tx_power from the CONFIG_DB. The user input for TX power is validated here https://github.com/Azure/sonic-utilities/pull/2197.  Please do note, tx_power can be configured even if transceiver is NOT plugged in, so we cannot validate tx_power in that case.

Reply:
Is this the right place that it validates tx power https://github.com/prgeor/sonic-utilities/blob/ba58d985b2483d24fb68821f11c6507dd08de567/scripts/portconfig#L332? Does it mean it only allows one digit after decimal point? Does it verify the configured tx power is within the min and max advertised supported tx power values?